Src/Plugins/Tests.Security/Helpers/HttpMockHelper.cs

```diff
@@ -22,6 +22,8 @@ public class HttpMockHelper : DelegatingHandler
             new HttpResponseMessage(HttpStatusCode.OK);
         public static readonly HttpResponseMessage NotFoundResponse =
             new HttpResponseMessage(HttpStatusCode.NotFound);
+        public static readonly HttpResponseMessage BadRequestResponse =
+            new HttpResponseMessage(HttpStatusCode.BadRequest);
```

Comment:
Suggest also adding `Forbidden` and `InternalServerError` for use in other tests.

Reply:
done